app/models/user.rb

```diff
@@ -491,10 +491,6 @@ def self.find_by_mail(mail)
     where(['LOWER(mail) = ?', mail.to_s.downcase]).first
   end
 
-  def self.find_all_by_mails(mails)
```

Comment:
I've checked the plugins I have cloned on my machine and let @ulferts check for occurences on his machine. Using GitHub I could only find rather old occurences in various observer classes, but they all looked like they lived in the (earlier) core.